frontend/src/use-cases/songs/screens/create-song/CreateSong.screen.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { useGammaStatus } from "@cthit/react-digit-components";
+import useAdmin from "../../../../common/hooks/use-admin";
+import InsufficientAccess from "../../../../common/components/insufficient-access";
+import CreateSongForm from "./components/create-song-form";
+import FormatSongInstructions from "../../components/format-song-instruction";
+import SongFormContainer from "../../components/song-form-container";
+
+const CreateSong = () => {
+    const [loading] = useGammaStatus();
+    const admin = useAdmin();
+
+    if (!loading && !admin) {
+        return <InsufficientAccess />;
+    }
+
+    return (
+        <SongFormContainer>
+            <div />
```

Comment:
why?

Reply:
SongFormContainer has a CSS grid with three columns so that the form can be in the middle and the format intructions to the right. But the first column needed some value so it got an empty div. It may not be a pretty solution but it did the job